x-pack/legacy/plugins/ml/public/data_visualizer/data_loader/data_loader.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
+import { decorateQuery, luceneStringToDsl } from '@kbn/es-query';
+import { i18n } from '@kbn/i18n';
+
+import { toastNotifications } from 'ui/notify';
+import { IndexPattern } from 'ui/index_patterns';
+
+import { SavedSearchQuery } from '../../contexts/kibana';
+import { IndexPatternTitle } from '../../../common/types/kibana';
+
+import { ml } from '../../services/ml_api_service';
+import { FieldRequestConfig } from '../common';
+
+export class DataLoader {
+  // List of system fields we don't want to display.
+  public static readonly OMIT_FIELDS: string[] = [
+    '_source',
+    '_type',
+    '_index',
+    '_id',
+    '_version',
+    '_score',
+  ];
+
+  public static MAX_EXAMPLES_DEFAULT: number = 10;
```

Comment:
`MAX_EXAMPLES_DEFAULT` is not used outside this class and so don't need to be public or a member of the class.

Reply:
Done in https://github.com/elastic/kibana/pull/42685/commits/992c67e94e727b7b5b5f91cbc73eb954c6e13a0f